### DISCUSSION TITLE
Backup from Bubble to Supabase

### DISCUSSION THREAD

**Discussion by nachotkn** [supabase/supabase#10852] [General]
Hello,

We’re exploring again Supabase since as we’re no programmers it’s kind of hard for us to make it works so I was wondering if you could help us with this:

Our goal is to make a backup of our Bubble database into Supabase every single week. So Supabase should have to check the modified records and the new one, insert the new ones and modify the files existing ones.

You can access to Bubble database via the Data API so Supabase can connect to the database and make the magic 😊. The problem is that we don’t know how to do it.

I’ve seen the Kanye api tutorial you have several time a but still no clue.

Do you mind helping us a bit so we can validate what we’re trying to accomplish?

At the end what we want is to make those database backup of 15 tables, about 20.000 records for each table that keeps growing so we can move everything to Supabase in a year or so, once we validate what we can do with Bubble and Supabase.

Thanks a lot. 

Looking forward to hearing from you.

Have a good day.

**Comment by wapdat** [comment]
The way I did it was to use the automatic export functionality in PlanBbackups.io 

Optionally it can create the tables or you can map the bubble tables to existing Supabase tables. You can setup daily syncing of the Bubble data to your Supabase tables also.
